src/views/InteractionDesigner.vue

```diff
@@ -251,7 +253,8 @@ export default {
     async registerBlockTypes() {
       const {blockClasses} = this
 
-      forEach(blockClasses, async (blockClass) => {
+      const blockInstallers = keys(blockClasses).map(async (blockClassKey) => {
+        const blockClass = blockClasses[blockClassKey]
```

Comment:
Minor suggestion. Would it work if we did it without keys? Like this:
```
const blockInstallers = blockClasses.map(async (blockClass) => {
   const type = blockClass.type
   ...
}
```

Reply:
Ah, true

Reply:
But that would save only one line at best, so I don't insist 😄

Reply:
It's a good suggestion! Much more straightforward.